guides/common/modules/proc_editing-a-resource-quota.adoc

```diff
@@ -2,9 +2,35 @@
 = Editing a resource quota
 
 You can edit a resource quota to adjust available resources to {Project} users.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-editing-a-resource-quota[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Configure* > *Resource Quotas*.
 . Select your resource quota.
 . Adjust the *CPU cores*, *Memory*, or *Disk space* and click *Apply* to submit your changes to {Project}.
 . Click *Create resource quota* to submit the resource quota to {Project}.
+
+[id="cli-editing-a-resource-quota"]
+.CLI procedure
+. Optional: List all resource quotas:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota list --fields id,name
+----
+. Edit a resource quota:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota update \
+--cpu-cores _My_CPU_Cores_ \
+--disk-space _My_Disk_Space_in_GiB_ \
+--id _My_Resource_Quota_ID_ \
+--memory _My_Memory_in_MiB_ \
```

Comment:
Are you absolutely sure that the unit is MebiBytes (MiB) and not MegaBytes (MB)? [Those are different units.](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units) And Mi- Gi- aren't usually used. :woman_shrugging: 

The same for the GiB above and other places.

Reply:
@bastian-src Can you please confirm if it's MiB or MR etc?

Reply:
Confirming that we're using MiB and GiB in Foreman Resource Quota.

We used Kubernetes as a reference which uses GiB as you can see [here](https://kubernetes.io/docs/concepts/policy/resource-quotas/#storage-resource-quota).